eng/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```diff
@@ -22,34 +22,38 @@ jobs:
   steps:
     - ${{ each config in parameters.MatrixConfigs }}:
       - ${{ if eq(config.GenerateVMJobs, 'true') }}:
-        - pwsh: |
-            $extraFilters = "${{ join('","', parameters.MatrixFilters) }}"
-            $filters = ,"container=^$","SupportedClouds=^$|${{ parameters.CloudConfig.Cloud }}"
-            if ($extraFilters) { $filters = @($extraFilters) + $filters }
-            Write-Host "DISPLAY NAME FILTER: `"$(displayNameFilter)`"; FILTERS: `"$filters`""
-            eng/scripts/job-matrix/Create-JobMatrix.ps1 `
-              -ConfigPath ${{ config.Path }} `
-              -Selection ${{ config.Selection }} `
-              -DisplayNameFilter "$(displayNameFilter)" `
-              -Filters $filters `
-              -NonSparseParameters "${{ join('","', config.NonSparseParameters) }}"
+        - task: Powershell@2
+          inputs:
+            pwsh: true
+            targetType: inline
+            script: |
+              $VerbosePreference = 'Continue'
+              $filters = "${{ join('","', parameters.MatrixFilters) }}", "container=^$", "SupportedClouds=^$|${{ parameters.CloudConfig.Cloud }}"
+              eng/scripts/job-matrix/Create-JobMatrix.ps1 `
+                -ConfigPath ${{ config.Path }} `
+                -Selection ${{ config.Selection }} `
+                -DisplayNameFilter "$(displayNameFilter)" `
+                -Filters $filters `
+                -NonSparseParameters "${{ join('","', config.NonSparseParameters) }}"
+          displayName: Generate VM Job Matrix ${{ config.Name }}
           name: generate_vm_job_matrix_${{ config.Name }}
-          displayName: Generate VM Job Matrix
 
       - ${{ if eq(config.GenerateContainerJobs, 'true') }}:
-        - pwsh: |
-            $extraFilters = "${{ join('","', parameters.MatrixFilters) }}"
-            $filters = ,"container=.*","SupportedClouds=^$|${{ parameters.CloudConfig.Cloud }}"
-            if ($extraFilters) { $filters = @($extraFilters) + $filters }
-            Write-Host "DISPLAY NAME FILTER: `"$(displayNameFilter)`"; FILTERS: `"$filters`""
-            eng/scripts/job-matrix/Create-JobMatrix.ps1 `
-              -ConfigPath ${{ config.Path }} `
-              -Selection ${{ config.Selection }} `
-              -DisplayNameFilter "$(displayNameFilter)" `
-              -Filters $filters `
-              -NonSparseParameters "${{ join('","', config.NonSparseParameters) }}"
-          name: generate_container_job_matrix_${{ config.Name }}
+        - task: Powershell@2
+          inputs:
+            pwsh: true
+            targetType: inline
+            script: |
+              $VerbosePreference = 'Continue'
```

Comment:
If you want verbose logs you can also pass the common `-Vebose` option to enable it for the script call.